zkevm-circuits/src/pi_circuit.rs

```diff
@@ -311,6 +342,11 @@ pub struct PiCircuitConfig<F: Field> {
     tx_table: TxTable,
     keccak_table: KeccakTable,
 
+    l1_block_hashes_count: Column<Advice>, // RLC(rpi) as the input to Keccak table
```

Comment:
Also here, comments are welcome. The comment is not really verbose to understand what's happening. The PI circuit is heavily under-documented in general. I would like to (soon) create a PR to change that. But in the meantime, I would like new additions to come with good documentation.